.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "AWS CDK & Python Development Environment",
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
+  "features": {
```

Comment:
can we add setup for pre-commit?  something like this.. https://github.com/Opentrons/vscode-dev-container-template/blob/main/.devcontainer/devcontainer.json#L41

Reply:
It's up to you. Feel free to suggest an update (using GitHub suggestion tool) to the `postCreateCommand` command below and I'll approve it.

Reply:
Is this the command you want to run automatically once the container has started?

```
pip install --user -U pre-commit && pre-commit install-hooks
```

Reply:
yes, it's for https://github.com/Sage-Bionetworks-IT/openchallenges/blob/dev/README.md#static-analysis